.goreleaser.yml

```
@@ -39,10 +39,12 @@ dockers:
   - goos: linux
     goarch: amd64
     image_templates:
-      - "ghcr.io/philips-labs/{{ .ProjectName }}:v{{ .Version }}"
-      - "ghcr.io/philips-labs/{{ .ProjectName }}:{{ .FullCommit }}"
+      - "ghcr.io/philips-labs/{{ .ProjectName }}:v{{ .Version }}-amd64"
+      - "ghcr.io/philips-labs/{{ .ProjectName }}:{{ .FullCommit }}-amd64"
+    use: buildx
     build_flag_templates:
       - "--pull"
+      - "--platform=linux/amd64"
       - "--label=org.opencontainers.image.created={{.Date}}"
       - "--label=org.opencontainers.image.title={{.ProjectName}}"
       - "--label=org.opencontainers.image.description={{.ProjectName}}"
```

We missed this one before.
Wrong location, better after revision, but I cannot add the suggestion there.
```suggestion
      - "--label=org.opencontainers.image.description={{.ProjectName}}"
      - "--label=org.opencontainers.image.source={{.GitURL}}"
```